app/assets/javascripts/bulk-tagger.js

```diff
@@ -112,14 +112,27 @@
       $content_item_forms.on(
         'ajax:success',
         function() {
+          display_response_state($(this), "Saved");
```

Comment:
Should this have a "Saving..." state also? How will users know that the 2nd or 3rd tag they enter has been saved?